packages/core/src/components/Form.js

```diff
@@ -131,6 +131,23 @@ export default class Form extends Component {
       schemaValidationErrors,
```

Comment:
Could we simplify this by just removing these two lines (then we would have no need for `getStateForEmit`)?

```suggestion
```

Reply:
Yes, thanks!